improver_tests/fire_weather/test_build_up_index.py

```diff
@@ -0,0 +1,366 @@
+# (C) Crown Copyright, Met Office. All rights reserved.
+#
+# This file is part of 'IMPROVER' and is released under the BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+import warnings
+
+import numpy as np
+import pytest
+from iris.cube import Cube, CubeList
+
+from improver.fire_weather.build_up_index import BuildUpIndex
+from improver_tests.fire_weather import make_cube, make_input_cubes
+
+
+def input_cubes(
+    dmc_val: float = 10.0,
+    dc_val: float = 15.0,
```

Comment:
I think you could use pytest fixtures to simplify these tests:
```suggestion
@pytest.fixture
def input_cubes(
    dmc_val: float,
    dc_val: float,
```